modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -64,6 +64,8 @@ export interface IExecuteProposalParams {
 export interface ICanVoteParams {
   pluginAddress: string;
   proposalId: string;
+  // TODO
+  // addressOrEns ??
```

Comment:
Yes, I left that there so I don't forget about it, this address can be an ens, from what I understand the contracts resolve ens addresses so all addresses should be `addressOrEns`, is this correct?

If it is correct should we validate this the ens address by checking if it has the correct format `something.whatever.eth` or should we check that the address resolves on the validation?

Reply:
1) If this is something we should do now, let's do it. If it is a task for later, let's have the task created. 

2) Yes, if we accept ENS domains, we definitely need to validate them, otherwise checks will fail